README.md

```diff
@@ -43,93 +43,94 @@ pip install dash-tools
 
 ## **Usage Examples**
 
-Below are common usage examples. For a more in-depth tutorial on writing apps for Plotly Dash, see the [Plotly Dash Documentation](https://dash.plotly.com/layout). For information about dash-tools commands, read the [_Commands_](#commands) section.
+### A. Create an App
 
-### **Creating A New Project**
-
-> _For a more in-depth tutorial on using dash-tools, check out the [Create and Upload Your App](/docs/Configuring-Your-App.md) document!_
-
-Creating a new Dash project is very simple. The following command will create a new directory called "MyDashApp":
+1. Create a Dash project in a new directory called "MyDashApp" (using your terminal or command prompt):
```

Comment:
Should we mention here that "MyDashApp" name can be changed to anything?

Reply:
yes, definitely. Just added it.